simulate-travis.py

```diff
@@ -257,6 +263,58 @@ def _set_channel_order():
     sp.check_call([bin_for('conda'), 'config', '--get'])
 
 
+def _travis_run(env):
+    if (
+        (env['TRAVIS_OS_NAME'] == 'linux') &
+        (not args.disable_docker) &
+        ('--docker' not in env['BIOCONDA_UTILS_BUILD_ARGS'])
+    ):
+        env['DOCKER_ARG'] = '--docker'
+        if not args.disable_mulled:
+            env['DOCKER_ARG'] += ' --mulled-test'
+
+    # Override env with whatever's in the shell environment
+    env.update(os.environ)
+
+    # Only modify the path just before running travis-run.sh:
+    if 'CONDA_ROOT' in os.environ:
+        env['PATH'] = os.path.join(
+            os.environ['CONDA_ROOT'], 'bin') + ':' + env['PATH']
+
+    try:
+        sp.check_call(['scripts/travis-run.sh'], env=env, universal_newlines=True)
+    except sp.CalledProcessError:
+        sys.exit(1)
+
+
+def _docker_machine_run_travis(env):
+    try:
+        p = sp.check_output(['docker-machine', 'env'])
+        os.environb.update({
```

Comment:
Just curious: what's the rationale for bytestrings here, rather than `universal_newlines=True` in the `check_output` call?